File: linear_search/C/linear search.c
#include <stdio.h>
void main()
{
    int a[10], i, item,n,flag=0;
    printf("Enter number of elements : ");
    scanf("%d",&n);
    printf("\nEnter elements of an array:\n");
    for (i=0; i<n; i++)
    {printf("Element : ");
        scanf("%d", &a[i]);}
    printf("\nEnter item to search: ");
    scanf("%d", &item);
    for (i=0; i<n; i++)
       {
        if (item == a[i])
        {   flag=1;
            printf("\nItem found at location %d", i+1);
            break;
        }
       }
    if (flag==0)
        printf("\nItem does not exist.");
    getch();
}

File: linear_search/Python/linearsearch.py
print("************************Linear search*****************************")
arr=list()
n=input("Enter the number of elements : ")
for i in range(int(n)):
	ele=int(input("Element : "))
	arr.append(int(ele))
key=int(input("Enter the key(elemt to be searched): "))
flag=0
for i in range(int(n)):
	if(arr[i]==key):
		print("Element Found at ",  i+1)
		flag=1
if flag==0:
	print("Element not found ")
k=input("************************* Press Enter to Exit****************")

